src/cpp/src/continuous_batching_impl.cpp

```diff
@@ -55,10 +133,13 @@ void ContinuousBatchingPipeline::ContinuousBatchingImpl::initialize_pipeline(
     std::shared_ptr<ov::Model> model,
     const SchedulerConfig& scheduler_config,
     const ov::AnyMap& properties,
-    const DeviceConfig& device_config,
-    ov::Core& core) {
+    const DeviceConfig& device_config) {
+    ov::Core core = utils::singleton_core();
     ov::CompiledModel compiled_model;
 
+    // TODO: remove once plugin automatically set KV cache precisions
+    apply_kv_cache_precision(model, device_config.get_device(), properties);
```

Comment:
@luo-cheng2021 @sshlyapn 
This is current WA for this branch to override that `undefined` precisions to values guessed within `apply_kv_cache_precision`

It's supposed that once CPU / GPU can compile `undefined` precisions and set proper types (as in SPDA), we can drop this function at all.